server/events/models/models.go

```diff
@@ -435,6 +435,11 @@ func (p ProjectCommandContext) GetShowResultFileName() string {
 	return fmt.Sprintf("%s-%s.json", projName, p.Workspace)
 }
 
+// Gets a unique identifier for the current pull request as a single string
+func (p *ProjectCommandContext) PullInfo() string {
```

Comment:
To keep things consistent let make this change.
```suggestion
func (p ProjectCommandContext) PullInfo() string {
```